src/audio-view.js

```diff
@@ -0,0 +1,193 @@
+/*
+  Game event               Sound                      Logic
+
+  Enter alone mode         Alone_Mode/Pulse_amb_loop  FIXME
+  Handshake                Alone_Mode/Hand_Shake_01   changes.handshakes (Set of bool)
+
+Mole:
+  Panel activated          Game_01/G01_LED_XX
+  Active panel touched     Game_01/G01_Success_01
+  Non-active panel touched Game_01/G01_Negative_01
+
+Disk:
+
+Simon:
+  Panel activated during pattern animation
+                           Game_03/G03_LED_XX
+  Correct panel touched
+                           Game_03/G03_LED_XX
+  Wrong panel touched
+                           Game_03/G03_Negative_01
+  Won level (after short delay)
+                           Game_03/G03_Success_01
+  Won all levels (after short delay)
+                           Game_03/G03_Light_Show_01
+
+*/
+
+const _ = require('lodash');
+
+const SculptureStore = require('@anyware/game-logic/lib/sculpture-store');
+const GAMES = require('@anyware/game-logic/lib/constants/games');
+import {Sound, VCFSound} from './audio-api';
+
+export default class AudioView {
+  constructor(store, config, dispatcher) {
+    this.store = store;
+    this.config = config;
+  }
+
+  // Loads all sounds, calls callback([err]) when done
+  load(callback) {
+    // Maps logical sound identifiers to filenames. We'll load these sounds next.
+    this.sounds = {
```

Comment:
In that case, should we have a sound-specific config in this repository? This config depends on the audio API. I feel this needs to be addressed as technical debt once we have a more clear understanding of our final audio requirements.